multiplayer/src/epics/resumeGameAsync.ts

```diff
@@ -13,6 +14,18 @@ export async function resumeGameAsync() {
         }
         simDriver()?.resume(pxsim.SimulatorDebuggerCommand.Resume);
         dispatch(setGamePaused(false));
+        if (clientRole === "host") {
+            for (let i = 1; i <= 4; i++) {
+                const nowConnected = !!presence.users.find(el => el.slot === i);
+                simDriver()?.postMessage({
+                    type: "multiplayer",
+                    content: "Connection",
+                    slot: i,
+                    connected: nowConnected,
+                } as SimMultiplayer.MultiplayerConnectionMessage);
+            }
+            await gameClient.sendCurrentScreenAsync();
```

Comment:
this is a quick 'side' fix, I noticed that the bug I was seeing before with xor diffs was occurring once in a while when switching tabs caused it to pause -- maybe the backend is rejecting a posted diff as it's already in the pause state or something. This felt like a pretty safe / reasonable place to put in a 'make sure everyone is synced now that we're resuming' either way